src/cmps/GamingCard.tsx

```diff
@@ -0,0 +1,76 @@
+import colors from "../assets/constants/colors";
+import {
+  StyledGamingCard,
+  ImageContainer,
+  AvatarContainer,
+  StyledProgress,
+  PowerLevelContainer,
+  PokemonImage,
+} from "../styles/StyledGamingCard";
+import Typography from "../styles/Typography";
+import { Pokemon } from "../interfaces/Pokemon";
+import { PowerLevel } from "../styles/StyledPokemonCard";
+import { SPACING } from "../assets/constants/spacings";
+
+interface GamingCardProps {
+  pokemon: Pokemon | null;
```

Comment:
If the pokemon is null is there an empty state?
cause if not it should manadatory